recipes/eastl/all/conandata.yml

```diff
@@ -21,20 +24,51 @@ sources:
     url: "https://github.com/electronicarts/EASTL/archive/3.15.00.tar.gz"
     sha256: "1578a2935ca490920b96c960fb570dab025280bb978fba40c88eb9ac74760c14"
 patches:
+  "3.21.12":
+    - patch_file: "patches/3.21.12-0001-cmake-shared-use-conan-add-install.patch"
+      patch_description: "support shared build, use cci's EABase, add install definitions"
+      patch_type: "conan"
+    - patch_file: "patches/3.17.03-0002-revert-c++14-constexpr.patch"
+      patch_description: "disable constexpr for unsupported compilers"
+      patch_type: "portability"
+    - patch_file: "patches/3.21.12-0003-rename_reference.patch"
```

Comment:
[The linked PR](https://github.com/electronicarts/EASTL/pull/513/files) has more contents in the patch than what is in this file, should we use the whole diff instead of just the one that was making it fail to compile?

Reply:
@RubenRBS 
I think that no further modifications are necessary.
This is because the other source codes (array.h, span.h) have been modified since 3.21.12.
https://github.com/electronicarts/EASTL/blob/3.21.12/include/EASTL/array.h
https://github.com/electronicarts/EASTL/blob/3.21.12/include/EASTL/span.h

Reply:
Oki, thanks for the clarification :)

Let's wait until we get a response in https://github.com/electronicarts/EASTL/issues/512#issuecomment-1623476892 to clarify the better way forward